book/src/remapping.md

```diff
@@ -53,4 +56,4 @@ Control, Shift and Alt modifiers are encoded respectively with the prefixes
 Keys can be disabled by binding them to the `no_op` command.
 
 Commands can be found at [Keymap](https://docs.helix-editor.com/keymap.html) Commands.
```

Comment:
Here, we should also add the link for TypeableCommands.

Reply:
Such as:
```markdown
Typeable Commands can be found at [Commands](https://docs.helix-editor.com/commands.html)
```

Reply:
Gâcher